### PR TITLE
basic game

### PR DESCRIPTION
**Pull request by Nikm2023**

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    x = int(input())  #score player1
+    y = int(input())   #score player2
+    z = int(input())   #score player3
+    if(x > 8 or x < 1):   #invalid input
+        x = 0
+    if(y > 8 or y < 1):
+        y = 0
+    if(z > 8 or z < 1):
+        z = 0
+    if(x != y and x != z):   #if all three were different
```
Comment (mamadmr) [inline game.py:12]: this condition is wronge

Comment (mamadmr) [inline game.py]: The condition is wronge

**Comment by mamadmr** [comment]
test :)

**Comment by Nikm2023** [comment]
> test :)

shod

**Comment by mamadmr** [comment]
can you see the comments on your code?

File: game.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    if(x != y and y != z and x != z):   #if all three were different
+        scores[0] += 2 * x
+        scores[1] += 2 * y
+        scores[2] += 2 * z
+    else:
+        if(x == y and y != z):   #if just x and y were the same
+            scores[0] += int(x / 2)
+            scores[1] += int(y / 2)
+            scores[2] += 2 * z
+        else:
```
Comment (mamadmr) [inline game.py:21]: use elif

File: game.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            else:
+                if(y == z and z != x):  #if just y and z were the same
+                    scores[1] += int(y / 2) 
+                    scores[2] += int(z / 2)
+                    scores[0] += 2 * x 
+                else:    #if all of them were the same
+                    scores[0] += int(x / 3)
+                    scores[1] += int(y / 3)
+                    scores[2] += int(z / 3)
+    print(f"p1-{x}:{scores[0]}\np2-{y}:{scores[1]}\np3-{z}:{scores[2]}")
```
Comment (mamadmr) [inline game.py:35]: it's too complex use three different if

File: game.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                scores[0] += int(x / 2)
+                scores[2] += int(z / 2)
+                scores[1] += 2 * y
+            else:
+                if(y == z and z != x):  #if just y and z were the same
+                    scores[1] += int(y / 2) 
+                    scores[2] += int(z / 2)
+                    scores[0] += 2 * x 
+                else:    #if all of them were the same
+                    scores[0] += int(x / 3)
```
Comment (mamadmr) [inline game.py:32]: use "a//b" instead of int(a/b)

File: game.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
+scores = [0, 0, 0]   #all scores in a row
+while(scores[0] < 101 and scores[1] < 101 and scores[2] < 101):
+    x = int(input())  #score player1
```
Comment (mamadmr) [inline game.py:3]: input is not the score of players

Comment (mamadmr) [reply game.py:3]: it's theire chosen number

File: game.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        y = 0
+    if(z > 8 or z < 1):
+        z = 0
+    if(x != y and y != z and x != z):   #if all three were different
+        scores[0] += 2 * x
+        scores[1] += 2 * y
+        scores[2] += 2 * z
+    elif (x == y and y != z): #if just x and y were the same
+        scores[0] += x // 2
+        scores[1] += y
```
Comment (mamadmr) [inline game.py:18]: :(((((((((((((((

File: game.py (hunk 5 above)
Comment (Nikm2023) [reply game.py:3]: comment is the problem?

File: game.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    if(x != y and y != z and x != z):   #if all three were different
+        scores[0] += 2 * x
+        scores[1] += 2 * y
+        scores[2] += 2 * z
+    elif (x == y and y != z): #if just x and y were the same
+        scores[0] += x // 2
+        scores[1] += y 
+        scores[2] += 2 * z
+    elif (x == z and z != y):  #if just x and z were the same
+        scores[0] += x // 2
```
Comment (mamadmr) [inline game.py:21]: sort assigning values by index

first assigne score[0] then score[1] and at the end score[2]

File: game.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        scores[2] += z // 2
+    elif (y == z and z != x):   #if just y and z were the same
+        scores[0] += 2 * x 
+        scores[1] += y // 2 
+        scores[2] += z // 2
+    else:    #if all of them were the same
+        scores[0] += x // 3
+        scores[1] += y // 3
+        scores[2] += z // 3
+    print(f"p1-{x}:{scores[0]}\n")
```
Comment (mamadmr) [inline game.py:32]: check the notion 
the format should be like 
p1:{x}:score[0]
I deleted '-'

Comment (mamadmr) [reply game.py:32]: I split them with ':'

**Pull request merged**